Tests/fixtures/smartphone-34.yml

```diff
@@ -8051,24 +8035,6 @@
     model: C2 Tennen
   os_family: Android
   browser_family: Chrome
--
-  user_agent: Mozilla/5.0 (Linux; Android 7.0; SNAP_PRO) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/76.0.3809.132 Mobile Safari/537.36
-  os:
-    name: Android
-    version: "7.0"
-    platform: ""
-  client:
-    type: browser
-    name: Chrome Mobile
-    version: 76.0.3809.132
-    engine: Blink
-    engine_version: 76.0.3809.132
-  device:
-    type: tablet
```

Comment:
https://myghmarket.com/product/nasco-snap-pro-dual-sim-16gb-hdd-1gb-ram/

the smartphone